backend/src/main/java/ca/gov/dtsstn/vacman/api/event/listener/ProfileEventListener.java

```
@@ -78,5 +91,38 @@ public void handleProfileStatusChange(ProfileStatusChangeEvent event) throws Jso
 
 		log.info("Event: profile status changed - ID: {}, from status ID: {}, to status ID: {}", 
 			event.entity().getId(), event.previousStatusId(), event.newStatusId());
+
+		// Check if the profile status was changed to APPROVED
+		ProfileEntity profile = event.entity();
+		ProfileStatusEntity newStatus = profile.getProfileStatus();
+
+		if (newStatus != null && AppConstants.ProfileStatusCodes.APPROVED.equals(newStatus.getCode())) {
+			sendApprovalNotification(profile);
+		}
+	}
+
+	private void sendApprovalNotification(ProfileEntity profile) {
+		try {
+			String email = profile.getUser().getBusinessEmailAddress();
+
+			if (email != null && !email.isEmpty()) {
+				String profileId = profile.getId().toString();
+				String username = profile.getUser().getFirstName() + " " + profile.getUser().getLastName();
+
+				// Send the notification
+				notificationService.sendEmailNotification(
+					email, 
+					profileId, 
+					username, 
+					ProfileStatus.APPROVED
+				);
+
+				log.info("Approval notification sent to user with email: {}, profile ID: {}", email, profileId);
+			} else {
+				log.warn("Could not send approval notification - no email address found for profile ID: {}", profile.getId());
+			}
+		} catch (Exception e) {
+			log.error("Error sending approval notification for profile ID: {}", profile.getId(), e);
+		}
```

Since this is executed asynchronously (once correctly configured), you don't have to try/catch.. Just let the exception bubble up through the stack and the framework will automatically log it for you in the global exception handler.

---

yes that makes sense, removed the try/catch